tests/arrays/arrays.cpp

```diff
@@ -623,6 +636,9 @@ TEST_CASE("wxDynArray::" #name, "[dynarray]")                                 \
     CHECK( b.Index( 5 ) == 2 );                                      \
     CHECK( b.Index( 6 ) == wxNOT_FOUND );                            \
     CHECK( b.Index( 17 ) == 3 );                                     \
+                                                                     \
+    wxArray##name c({1,2,3});                                        \
```

Comment:
I imagine I need to duplicate this big macro as well (and this is the reason why the tests are failing).
Any other suggestion?

Reply:
Again, although it's still not going to be pretty, adding `WX_TEST_INIT_LIST` helper macro defined as non-empty only if init list is available would make this not too horrible.